ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -197,3 +197,7 @@ tests:
   commands: TEST_SUITE=openshift/conformance/parallel run-tests
   openshift_installer_upi:
     cluster_profile: vsphere
+- as: e2e-steps
```

Comment:
This should probably include `aws` and `ipi` somewhere in its name, since the installer is also going to want to run other flavors.

Reply:
The ultimate goal is to replace the existing job, so this is just a temporary name (see also the previous response).